Classes/Notifications/NotificationsViewController.swift

```diff
@@ -119,6 +123,8 @@ NotificationNextPageSectionControllerDelegate {
         )
         feed.finishLoading(dismissRefresh: dismissRefresh, animated: animated)
         updateMarkAllEnabled()
+        
+        showDetailViewController(SplitPlaceholderViewController(), sender: self)
```

Comment:
Ohh true.. My bad.
To be honest the implementation you've done with placeholder isn't quite right so I was trying to work within your constraints.

Normally your detail view controller is defined in the storyboard and you use the splitViewDelegate to override or wateva for things like an empty controller.

Sorry about that.